source/extensions/tracers/xray/daemon_broker.cc

```diff
@@ -40,11 +36,9 @@ void DaemonBrokerImpl::send(const std::string& data) const {
   constexpr auto version = 1;
   constexpr auto format = "json";
   const std::string payload = absl::StrCat(createHeader(format, version), "\n", data);
-  Buffer::RawSlice buf;
-  buf.mem_ = const_cast<char*>(payload.data());
-  buf.len_ = payload.length();
-  const auto rc = Network::Utility::writeToSocket(*io_handle_, &buf, 1 /*num_slices*/,
-                                                  nullptr /*local_ip*/, *address_);
+  Buffer::OwnedImpl buf(payload);
```

Comment:
Note this will copy which might not be what you want.

Reply:
ugh - yeah. I'll revert that.